internal/utils/utils.go

```diff
@@ -0,0 +1,54 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"syscall"
+
+	p2pms "github.com/statechannels/go-nitro/node/engine/messageservice/p2p-message-service"
+)
+
+// WaitForPeerInfoExchange waits for all the P2PMessageServices to receive peer info from each other
+func WaitForPeerInfoExchange(services ...*p2pms.P2PMessageService) {
+	for sNum, s := range services {
+		for i := 0; i < len(services)-1; i++ {
+			peerInfo := <-s.PeerInfoReceived()
+			fmt.Printf("Service num: %d, peer num: %d, peerInfo: %v\n", sNum, i, peerInfo)
+		}
+	}
+}
+
+// StopCommands stops the given executing commands
+func StopCommands(cmds ...*exec.Cmd) {
```

Comment:
I think there may be a rebase/merge issue here? I think a bunch of these util functions were refactored into [start-rpc-servers](https://github.com/statechannels/go-nitro/blob/a90cdb294aaa1902f93db15ed612d393253cee03/cmd/start-rpc-servers/main.go#L205)

Reply:
I'm not really sure what happened. I tried to pull in the latest changes using these commands:
```
git checkout main
git pull origin main
git checkout kademlia-dht
git merge main
```
Then I addressed all of the merge conflicts and ran these commands.
```
git add -A
git commit
```
Any idea what might have gone wrong in that sequence?

Reply:
Maybe it was caused because the entire files were deleted on `main`. I removed the files `internal/utils` and `internal/nitro-network` again on this branch so it should be good now

Reply:
I generally `rebase` instead of `merge`, IE:`git fetch origin && git rebase origin/main` but what you did makes sense to me.

>Then I addressed all of the merge conflicts and ran these commands.

I suspect this is where things might have gone wrong. When resolving merge conflicts it's easy to include a file that you've modified that's been removed in `main`.